ts/Series/Pie/PiePoint.ts

```diff
@@ -181,11 +181,19 @@ class PiePoint extends Point {
 
     /**
      * Toggle the visibility of the pie slice.
-     * @private
+     * Note that this method is available only for `pie` series.
```

Comment:
Not only pies. Other series like funnel, treemap, timeline etc. also implement it. It is probably safest to say "available only for some series, like pie, treemap and sunburst". Then don't exclude anything.